roles/kubernetes/master/defaults/main.yml

```diff
@@ -60,8 +60,8 @@ kube_oidc_auth: false
 # kube_oidc_groups_claim: groups
 
 ##Variables for custom flags
-apiserver_custom_flags: []
+apiserver_custom_flags: ['--kubelet-preferred-address-types=InternalDNS,InternalIP,Hostname,ExternalDNS,ExternalIP']
```

Comment:
This should be a new variable, so that a user can add a new custom flag simply without having to repeat this logic.

Please put this line back and make a new line:

```
kube_preferred_address_types: 'InternalDNS,InternalIP,Hostname,ExternalDNS,ExternalIP'
```

And then modify the the template for kube-apiserver config to add specify this param